src/test/regress/expected/multi_colocation_utils.out

```diff
@@ -944,40 +943,53 @@ SELECT * FROM pg_dist_colocation
     ORDER BY colocationid;
  colocationid | shardcount | replicationfactor | distributioncolumntype | distributioncolumncollation
 ---------------------------------------------------------------------
-            2 |          2 |                 1 |                     23 |                           0
-            3 |          2 |                 2 |                     25 |                         100
-            4 |          8 |                 2 |                     23 |                           0
+            1 |          2 |                 1 |                     23 |                           0
+            2 |          2 |                 2 |                     25 |                         100
+            3 |          8 |                 2 |                     23 |                           0
+            4 |          2 |                 2 |                     23 |                           0
```

Comment:
@marcocitus do you think it is okay to have the same row for different colocation ids here? Should we make sure that we always return the smallest colocation id in this case in case of tie?

Reply:
Before merging, you can ask Marco if this part is okay.

Reply:
Hmz, do we know why we did not see this row before? (shard count 2, replication factor 2, column type int)

Reply:
Possibly because we removed this command:

SELECT mark_tables_colocated('table1_groupF', ARRAY['table2_groupF']);

But even if we put that command, it is probably expected that there will be two rows where the only difference is the colocation id. That might be a concern for another PR, I just thought we might want to deterministically return the colocation id when this happens so that by default all table creations go to the same colocation group.

Reply:
I might be missing something, but all these rows look unique to me.

Reply:
All the rows are unique.

I was curious if we had to do anything special for the case where there are multiple rows with the same <shardCount, replicationFactor, distributionColumnType,distributionColumnColocation>. 

For example in this output colocation id 4 and colocation id 5 have the same  <shardCount, replicationFactor, distributionColumnType,distributionColumnColocation>. And pg11 returns colocation id 5, whereas pg12 and pg13 returns colocation id 4 from `ColocationId` method. 

Hence it seems like there is no guarantee  which colocation id will be returned when there are multiple matches from ColocationId method, which could mean that creating 2 distributed tables with default colocation id could go into different colocation groups.

Reply:
https://github.com/citusdata/citus/pull/4559